lib/internal/cluster/primary.js

```diff
@@ -45,6 +45,7 @@ let ids = 0;
 let initialized = false;
 
 // XXX(bnoordhuis) Fold cluster.schedulingPolicy into cluster.settings?
+// XXX(alexfernandez) Folded cluster.schedulingPolicy into cluster.settings.
```

Comment:
Remove ben's comment instead of adding one?

Reply:
Done in 6617c78014.

Reply:
How about removing it?

Reply:
Done in commit af4d3deec0.